README.md

```diff
@@ -193,6 +193,13 @@ pass inferior arguments via `-a`:
     $ PYTHONPATH=$PWD scripts/myunit -m mypy.test.testlex -v '*backslash*'
     $ ./runtests.py mypy.test.testlex -a -v -a '*backslash*'
 
+Mypy is currently in the process of converting its tests from myunit to pytest.
+Some of the tests, such as `test_check`, have already been converted. To run
+these, you must use pytest instead, and use `-k` instead of `-a`:
+
+    $ ./runtests.py pytest -t -k -t NestedListAssignment
+    $ py.test -k NestedListAssignment
+
```

Comment:
"To run these, you must use pytest instead", sounds like these tests might not run if you just say `./runtests.py`. They do, right?

Maybe a clearer description here would be something like "To run these unit tests individually, you can run `py.test` directly, or pass inferior arguments with `-t`." (I think you mean `-t` rather than `-k` -- is that right or am I misreading the example?)